### ISSUE TITLE
Uninstaller should have an option to remove vms and images on Windows

### ISSUE THREAD

**Issue by jecisc** [pharo-project/pharo-launcher#236]
The current uninstaller on Windows deletes the content of PharoLauncher directory. It should also propose as an option to delete the images and the vms downloaded via the launcher if the user desire to totally remove Pharo.

**Comment by demarey** [comment]
This option is really dangerous. I'm not sure I would like to have it for PharoLauncher.
By example, does VirtualBox proposes you to remove all the disks and virtual machines you created when uninstalling the app?

**Comment by jecisc** [comment]
I would like to have it with a big warning if possible.
For example imagine that someone wants to try Pharo. He install the Pharo launcher, play a little with Pharo and see that it does not fit he's needs. He will uninstall it and he will expect everything to be uninstalled, bacause he will never use it anymore.
I do not really care about this option since I will use Pharo for a looooong time hopefully, but for this kind of users it would be nice I think.

**Comment by VincentBlondeau** [comment]
I agree to have at least a button to reset the VMs that the new ones are downloaded. I am however not sure about the images.. It seems very dangerous for me.
